frontend/docs/code-conventions.md

```diff
@@ -0,0 +1,211 @@
+# Naming Conventions
+
+## Files and Folders
+- Use PascalCase for component files: `UserProfile.tsx`
+- Use kebab-case for non-component files: `auth-utils.ts`
+- Use plural for folder names: `components/`, `utils/`
+- Name page components with "Page" suffix: `HomePage.tsx`, `UserProfilePage.tsx`
+- Name layout components with "Layout" suffix: `MainLayout.tsx`
```

Comment:
@francisl PascalCase for component folder name?

ex:
QueryBuilder/
   QueryBar.tsx
   QueryCombiner.tsx
   ...

Reply:
j'ai clarifié la docs.

Ton exemple est bon.